drivers/input/Kconfig.sdl

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) 2020 Jabil Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-config KSCAN_SDL
+config INPUT_SDL_TOUCH
```

Comment:
Please add a note to doc/releases/release-notes-3.4.rst about Kconfig option KSCAN_SDL.

Reply:
Hey @jfischer-no I'm preparing a separate PR with all the input related changes release notes, you mind if I do this separately?